testing/iframe.js

```diff
@@ -296,10 +299,17 @@ export function createIframeWithMessageStub(win) {
   element.expectMessageFromParent = msg => {
     return new Promise(resolve => {
       const listener = event => {
+        let expectMsg = msg;
+        let eventMsg = event.data.receivedMessage;
+        if (typeof expectMsg !== 'string') {
+          expectMsg = JSON.stringify(expectMsg);
+        }
+        if (typeof eventMsg !== 'string') {
```

Comment:
You mean `expectMsg` and `eventMsg` type are matched anyway, no need to check the type again?